### DISCUSSION TITLE
opening a pdf/png in Preview app in ios using maui

### DISCUSSION THREAD

**Discussion by Esal01** [dotnet/maui#16821] [Q&A]
Hi, a question from a beginner on Maui.
Is there an easy way to open/load a pdf/png file directly in the standard app on an IOS device like iPad?
I have tried Launcher but it only shows the sharing menu!

Thank you in advanced.

**Comment by baaaaif** [comment]
try https://devblogs.microsoft.com/xamarin/previewing-files-with-xamarin-ios/ 
this will work, I'm using it myself...
UIViewController -> Platform.GetCurrentUIViewController()

**Reply by Esal01** [reply]
Thank you I will try it.
